pom.xml

```diff
@@ -20,26 +20,21 @@
 
   <groupId>co.cask</groupId>
   <artifactId>dynamic-schema-sinks</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>1.0.0</version>
   <name>Dynamic Schema Sinks</name>
   <packaging>jar</packaging>
   <description>Dynamic schema based Sinks</description>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <!-- properties for script build step that creates the config files for the artifacts -->
-    <app.parents>system:system:cdap-data-pipeline[4.1.0,4.2.0-SNAPSHOT),system:cdap-data-streams[4.1.0,4.2.0-SNAPSHOT)</app.parents>
-    <cdap.version>4.1.0</cdap.version>
+    <cdap.version>4.3.1</cdap.version>
```

Comment:
make it 4.3.2?

Reply:
don't think its necessary since we've not really changed any apis, and this is build version only.